docs/attributes.md

```diff
@@ -147,6 +148,38 @@ The `Vector3` type is a struct composed of three `f32`s:
 
 A Vector3 with the value `10, 20, 30` looks like this: `00 00 20 41 00 00 a0 41 00 00 f0 41`.
 
+### CFrame
+**Type ID `0x14`**
+
+The `CFrame` type is composed of a `Vector3`, a 1-byte rotation ID, and an optional rotation matrix:
+
+| Field Name      | Format              | Value                                                                                                                                                                                                                                                  |
+|:----------------|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Position        | `Vector3`           | The positional component of the `CFrame`.                                                                                                                                                                                                              |
+| Rotation ID     | `u8`                | An identifier representing axis-alignment of the rotation, determining whether the rotation matrix is present (see `Rotation matrix`). If the rotation *is* axis-aligned, this field is one of the values from the table below; otherwise, it is `00`. |
+| Rotation matrix | `Optional<Matrix3>` | The rotational component of the `CFrame`, stored as a sequence of nine `f32`s. If Rotation ID is `00`, this field is the rotation matrix in **row-major order**, the XVector, the YVector, and ZVector, in that order; otherwise, it is absent.        |
```

Comment:
See my comment on the correction to the documentation for CFrame in the binary spec [here](https://github.com/rojo-rbx/rbx-dom/pull/297#discussion_r1242917009).